Library/Formula/sdl_ttf.rb

```diff
@@ -18,6 +18,10 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
```

Comment:
Backporting the patch to the original `sdl_ttf` is fine, I think - it is an official patch even if it's not in this branch of the codebase. Just to make sure, this is the exact same patch as in the sdl2_ttf version?

We have a number of packages using `sdl_ttf`, so I'd rather not remove it entirely.